src/Widgets/WindowSwitcher.vala

```diff
@@ -278,9 +278,7 @@ namespace Gala {
             float nat_width, nat_height;
             container.get_preferred_size (null, null, out nat_width, null);
 
-            if (container.get_n_children () == 1) {
-                nat_width -= WRAPPER_PADDING * scaling_factor;
-            }
+            nat_width -= (WRAPPER_PADDING * scaling_factor) / container.get_n_children ();
```

Comment:
Is a division by zero possible with the return of get n children here?

Reply:
It is not. Line [243](https://github.com/elementary/gala/blob/349c7b6843c54b6e077d2a730b1628637589e806/src/Widgets/WindowSwitcher.vala#L243) just above checks for a zero return on get_n_children and returns if that’s the count before we reach this statement.